metaflow/metaflow_config.py

```diff
@@ -315,7 +315,9 @@
 # Default port number to open on the pods
 KUBERNETES_PORT = from_conf("KUBERNETES_PORT", None)
 # Default kubernetes resource requests for CPU, memory and disk
-KUBERNETES_RESOURCES = from_conf("KUBERNETES_RESOURCES", "")
+KUBERNETES_CPU = from_conf("KUBERNETES_CPU", "")
+KUBERNETES_MEMORY = from_conf("KUBERNETES_MEMORY", "")
+KUBERNETES_DISK = from_conf("KUBERNETES_DISK", "")
```

Comment:
Empty string works as a default too, but just to stick to other defaults I'd change these to `None` instead.